spread.yaml

```diff
@@ -1022,9 +1032,8 @@ suites:
             distro_install_package snapd qemu qemu-utils genisoimage sshpass qemu-kvm cloud-image-utils ovmf kpartx xz-utils
             snap install ubuntu-image --classic
 
-            # TODO: replace with distro package once 2.45 is in stable
+            # Install the snapd built
             dpkg -i "$SPREAD_PATH"/../snapd_*.deb
-            snap install core --beta
```

Comment:
Why this change?

Reply:
This is because at some point we added that because it was needed a change that was just in edge and beta, but it is not required now. I left comments in other suites about that.